src/libs/antares/study/parts/common/cluster_list.cpp

```diff
@@ -320,35 +316,30 @@ template<class ClusterT>
 int ClusterList<ClusterT>::saveDataSeriesToFolder(const AnyString& folder) const
 {
     if (empty())
-        return 1;
+        return false;
```

Comment:
If we return **false** (which is better that returning **1**, as we once were), then let's return a **bool**